src/vulkan-renderer/gpu_info.cpp

```diff
@@ -11,259 +11,219 @@
 
 namespace inexor::vulkan_renderer {
 
-void VulkanGraphicsCardInfoViewer::print_driver_vulkan_version() {
-    // The version of the available Vulkan API is encoded as a 32 bit integer.
-    // https://vulkan.lunarg.com/doc/view/latest/windows/vkspec.html#extendingvulkan-coreversions-versionnumbers
+std::string GPU_Info::yes_or_no(const bool condition) const {
```

Comment:
Do we really need this, we can just print true/false instead of yes/no.

Reply:
Oh right.

Reply:
Done.